frontend/apps/web/src/components/system/Overview.tsx

```diff
@@ -1,16 +1,26 @@
-import React from 'react'
-import { CardLayout, ProgressBar, StackLayout, Text, Table } from '@filecoin/ui'
-import styled from 'styled-components'
-import { System } from '@filecoin/types'
-
 import { getButton } from '@/pages/tests'
-
+import { filterItems } from '@filecoin/core'
+import { System } from '@filecoin/types'
+import {
+  CardLayout,
+  ProgressBar,
+  SearchInput,
+  StackLayout,
+  Table,
+  Text,
+} from '@filecoin/ui'
+import React, { useState } from 'react'
+import styled from 'styled-components'
 interface Props {
   system: System
 }
 
 export const Overview: React.FC<Props> = ({ system }) => {
   const totalSubsystems = system.subsystems.length
+  const [searchTerm, setSearchTerm] = useState('')
+  const results = !searchTerm
+    ? system.subsystems
+    : filterItems(system.subsystems, searchTerm, 'name')
```

Comment:
This is fine for now, we'll still need a search that will cover all the strings not just the name.

Reply:
put a console.log in filterItems and see how much times will it triger, if for example you write just one letter in the search, and it triggers more than once. You need to put results in useState.

Reply:
useState is fine in this case, since we need different results when searchTerm changes. So results would be handled in useEffect.